spec/timezone.html

```diff
@@ -282,8 +302,7 @@ <h1>Abstract operations</h1>
       <h1>CreateTemporalTimeZoneFromStatic ( _constructor_, _identifier_ )</h1>
       <emu-alg>
         1. Assert: ! IsValidTimeZoneName(_identifier_) is *true*.
-        1. If ! IsConstructor(_constructor_) is *false*, then
-          1. Throw a *TypeError* exception.
+        1. If ! IsConstructor(_constructor_) is *false*, throw a *TypeError* exception.
```

Comment:
I slightly prefer the two-line form, but happy to pick either consistently.

Reply:
I'm personally fine with either, but I had heard that the single line form was to be preferred, thus this change.